docs/website/developers/committers/release-process/make-release-artifacts.md

```diff
@@ -54,5 +54,5 @@ Make a signed tag for this release, and then push the tag:
 
 {% highlight bash %}
 git tag -s -m "Tag release ${VERSION_NAME} release candidate ${RC_NUMBER}" apache-brooklyn-${VERSION_NAME}-rc${RC_NUMBER}
-git push origin apache-brooklyn-${VERSION_NAME}-rc${RC_NUMBER}
+git push apache apache-brooklyn-${VERSION_NAME}-rc${RC_NUMBER}
```

Comment:
The origin name depends on how the remote was added (or the repository originally cloned); but someone following the release instructions at https://brooklyn.incubator.apache.org/developers/committers/merging-contributed-code.html will have the upstream remote named `apache`, so this is the "right" change IMO.